src/compiler/diagnosticMessages.json

```diff
@@ -3310,6 +3310,10 @@
         "category": "Error",
         "code": 7034
     },
+    "Try `npm install @types/{0}` if it exists or adding `declare module '{0}'` in a separate file.": {

```

Comment:
I hope new users start with `excludes: ["node_modules"]` or something so that almost all files are included anyway. In any case, the tweaked version is up now.